### DISCUSSION TITLE
Building ssl models in espnet

### DISCUSSION THREAD

**Discussion by pratikkumar018** [espnet/espnet#4413] [Q&A] [answered]
HI Team,
I am trying to build ssl models like wav2vec2, wavlm, hubert in espnet from scratch. I tried looking and only found for hubert. Can anyone point out how to build these models.

**Comment by sw005320** [comment, marked as answer]
We only have the hubert pre-training. We can increase the priority of the development of such pre-training recipes upon requests.

**Reply by pratikkumar018** [reply]
Thanks for the info. We will be eagerly waiting for other ssl framework like wav2vec2 & wavlm.
